proto/kvrpcpb.proto

```diff
@@ -1095,6 +1095,9 @@ message TimeDetailV2 {
     uint64 kv_read_wall_time_ns = 4;
     // Total wall clock time spent on this RPC in TiKV .
     uint64 total_rpc_wall_time_ns = 5;
+
+    uint64 grpc_exec_time_ns = 6;
```

Comment:
```suggestion
    uint64 kv_grpc_pool_exec_time_ns = 6;
```

Reply:
please also add a comment for the new fields.